doc/release-notes.md

```diff
@@ -102,8 +102,15 @@ will trigger BIP 125 (replace-by-fee) opt-in. (#11413)
   option `-deprecatedrpc=banscore` is used. The `banscore` field will be fully
   removed in the next major release. (#19469)
 
-- The `testmempoolaccept` RPC returns `vsize` and a `fee` object with the `base` fee
-  if the transaction passes validation. (#19940)
+- User-configured maximum feerates are no longer considered network rules. The
+  `sendrawtransaction` error code for exceeding `maxfeerate` has been changed from
+  `-26` to `-25`. The error string has been changed from "absurdly-high-fee" to
+  "Fee exceeds maximum configured by user (e.g. -maxtxfee, maxfeerate)." The
+  `testmempoolaccept` RPC returns `max-fee-exceeded` rather than `absurdly-high-fee`
+  as the `reject-reason` if the transaction's feerate exceeds `maxfeerate`. (#19339)
```

Comment:
I think this section can be moved to "low level changes" --> RPC section

Reply:
Done